src/Lumper.UI/ViewModels/Shared/Entity/EntityLumpViewModel.cs

```diff
@@ -171,24 +171,31 @@ public void UpdateViewModelFromModel()
             // Property updates on EL -> update on ELVM
             foreach (EntityPropertyViewModel propVm in entVm.Value.Properties)
             {
-                // Checking both references and underlying values. Jobs should generally
-                // just create an entirely new property (so ref compare is what matters here)
-                // so entityproperty ctor logic runs, but doesn't hurt to test for value
-                // changes as well.
-                switch (propVm)
+                // Checking both references and underlying values. Jobs can either create
+                // an entirely new property (so ref compare), or they can modify the existing
+                // property (so value compare)
+                if (
+                    propVm is EntityPropertyStringViewModel { EntityProperty: Entity.EntityProperty<string> sM } sVm
```

Comment:
These variables should probably use more descriptive names (`strVm` / `strViewModel`). `sVm` is a bit difficult to interpret.

Reply:
Fair enough. I think using m/vm shorthand in some cases makes sense, but I agree that it's quite hard to parse here.

Reply:
Doing `strVm`, don't want a million line breaks and codebase references viewmodels/models so much I think shorthand is fair.